src/StatusQ/Core/StatusScrollView.qml

```diff
@@ -0,0 +1,20 @@
+import QtQuick 2.14
+import QtQuick.Templates 2.14 as T
+
+import StatusQ.Controls 0.1
+
+Flickable {
+    id: root
+
+    implicitWidth: contentWidth
+    implicitHeight: contentHeight
+    clip: true // NOTE: in Qt6 clip true will be default
+    contentWidth: contentItem.childrenRect.width
+    contentHeight: contentItem.childrenRect.height
+    boundsBehavior: Flickable.StopAtBounds
+    maximumFlickVelocity: 0
+    synchronousDrag: true
+
+    T.ScrollBar.horizontal: StatusScrollBar { enabled: root.contentWidth > root.width }
```

Comment:
@caybro maybe you can help me with this crutch? I tried to use `policy: T.ScrollBar.AsNeeded` but that does not work. But i remember it definitely worked using Qt 5.9

Reply:
Yeah it won't work with `T.ScrollBar`

Reply:
See below why

Reply:
Thanks!